cupy/_math/trigonometric.py

```diff
@@ -130,11 +135,26 @@ def unwrap(p, discont=numpy.pi, axis=-1):
     p = cupy.asarray(p)
     nd = p.ndim
     dd = sumprod.diff(p, axis=axis)
+    if discont is None:
+        discont = period/2
     slice1 = [slice(None, None)]*nd     # full slices
     slice1[axis] = slice(1, None)
     slice1 = tuple(slice1)
-    ph_correct = _unwrap_correct(dd, discont)
-    up = cupy.array(p, copy=True, dtype='d')
+    dtype = numpy.result_type(dd.dtype, period)
+    if numpy.core.numeric.issubdtype(dtype, numpy.core.numeric.integer):
```

Comment:
Let's use documented API :smile:
```suggestion
    if numpy.issubdtype(dtype, numpy.core.numeric.integer):
```